python/lsst/pipe/supertask/activator.py

```diff
@@ -383,7 +419,7 @@ def parse_and_run(cls):
                                       help='list modules available to this activator')
         parser_activator.add_argument('-lp', '--list_packages', action="store_true", default=False,
                                       help='list packages available to this activator, defined by'
-                                           'TASK_PACKAGES')
+                                           ' TASK_PACKAGES')
```

Comment:
For the other argument help strings that overflow the space goes on the previous line.